src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,72 @@
+import { useCallback, useMemo, useState } from 'react';
+import { SearchLink } from './SearchLink';
+import { useSearchParams } from 'react-router-dom';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sex = searchParams.get('sex') || '';
+  const centuries = useMemo(() => {
+    return searchParams.getAll('centuries') || [];
+  }, [searchParams]);
+
+  const [query, setQuery] = useState('');
+
+  const centuriesNumbers = useCallback(
+    (century: string) => {
+      return centuries.includes(century)
+        ? centuries.filter(curentCentury => curentCentury !== century)
+        : [...centuries, century];
+    },
+    [centuries],
+  );
```

Comment:
There is no sense to use useCallback if you don't pass function to the chilld component. If you want to memorize result of the function use useMemo instead